src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/address_book/index.yaml

```
@@ -0,0 +1,53 @@
+twig_hooks:
+    hooks:
+        'sylius_shop.account.address_book.index':
+            buttons:
+                template: '@SyliusShop/account/address_book/index/buttons.html.twig'
+            addresses:
+                template: '@SyliusShop/account/address_book/index/addresses.html.twig'
+
+        'sylius_shop.account.address_book.index.buttons':
+            add_address:
+                template: '@SyliusShop/account/address_book/index/buttons/add_address.html.twig'
+
+        'sylius_shop.account.address_book.index.addresses':
+            default:
+                template: '@SyliusShop/account/address_book/index/addresses/default.html.twig'
+            list:
+                template: '@SyliusShop/account/address_book/index/addresses/list.html.twig'
+
+        'sylius_shop.account.address_book.index.addresses.default':
+            badge:
+                template: '@SyliusShop/account/address_book/index/addresses/default/badge.html.twig'
+            content:
+                template: '@SyliusShop/account/address_book/index/addresses/default/content.html.twig'
+
+        'sylius_shop.account.address_book.index.addresses.default.content':
+            address:
+                template: '@SyliusShop/account/address_book/index/addresses/default/content/address.html.twig'
+            actions:
+                template: '@SyliusShop/account/address_book/index/addresses/default/content/actions.html.twig'
```

To be consistent with the list structure below, I would be for moving actions to this Twig Hook:
```suggestion
        'sylius_shop.account.address_book.index.addresses.default':
            badge:
                template: '@SyliusShop/account/address_book/index/addresses/default/badge.html.twig'
            content:
                template: '@SyliusShop/account/address_book/index/addresses/default/content.html.twig'
            actions:
                template: '@SyliusShop/account/address_book/index/addresses/default/actions.html.twig'

        'sylius_shop.account.address_book.index.addresses.default.content':
            address:
                template: '@SyliusShop/account/address_book/index/addresses/default/content/address.html.twig'
```